logs-shipper/src/main/java/jersey/rest/SearchResource.java

```diff
@@ -60,7 +73,7 @@ public Response search(@QueryParam("message") String message, @QueryParam("heade
         } catch (IOException e) {
             e.printStackTrace();
         }
-        return Response.status(HttpURLConnection.HTTP_BAD_REQUEST).entity("Failed request\n").build();
+        return Response.status(HttpURLConnection.HTTP_BAD_REQUEST).entity("Bad request").build();
     }
```

Comment:
The method has too much logic. Let's break it

Reply:
let me know if this is enough....